akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowJoinSpec.scala

```diff
@@ -18,8 +22,11 @@ class FlowJoinSpec extends AkkaSpec(ConfigFactory.parseString("akka.loglevel=INF
 
   implicit val materializer = ActorMaterializer(settings)
 
+  implicit val defaultPatience =
+    PatienceConfig(timeout = Span(2, Seconds), interval = Span(200, Millis))
```

Comment:
You can also say `PatienceConfig(2.seconds)` if you don’t care about the interval (which I usually don’t).